apps/console/webpack.config.ts

```diff
@@ -162,6 +162,15 @@ module.exports = (config: WebpackOptionsNormalized, context: NxWebpackContextInt
         config.plugins.splice(indexHtmlWebpackPluginIndex, 1);
     }
 
+    // Remove ForkTSCheckerWebpackPluginIndex plugin added by NX and add HtmlWebpackPlugin instead.
+    const forkTSCheckerWebpackPluginIndex: number = config.plugins.findIndex((plugin: WebpackPluginInstance) => {
+        return plugin.constructor.name === "ForkTsCheckerWebpackPlugin";
+    });
+
+    if (forkTSCheckerWebpackPluginIndex !== -1) {
+        config.plugins.splice(forkTSCheckerWebpackPluginIndex, 1);
+    }
+
```

Comment:
Let's remove this as there is a way to conditionally enable this through env file.

https://github.com/wso2/identity-apps/blob/master/apps/console/.env.local#L8